Cargo.toml

```diff
@@ -61,7 +61,7 @@ version = "1.0.98"
 features = ["derive"]
 
 [dependencies.tokio]
-version = "0.2.15"
+version = "0.2.18"
```

Comment:
I updated this when debugging the issue in case it was a bug in tokio. It didn't turn out to fix anything, but I figured we might as well still update it as new versions come out until we do a release.